tests/execution_spec.rs

```diff
@@ -995,13 +939,13 @@ async fn run_assert(
     spec: &ExecutionSpec,
     env: &HashMap<String, String>,
     request: &APIRequest,
-    config: &ConfigModule,
+    tailcall_executor: Result<TailcallExecutor, ValidationError<String>>,
```

Comment:
avoid passing values as result.

Reply:
as discussed, it's required because earlier the result conversion from ConfigModule to Blueprint was tested but now that happens internally so we just have final result.

Marking this as resolved.